### PR TITLE
[HuaXiang] Device Data Container - 10309

### PR DESCRIPTION
**Pull request by jpradocueva**
Submitted on behalf of the submitter

new pull request template

### CODE REVIEW

File: 10309.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2018 Hua Xiang IOT. All rights reserved.
+
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>DeviceDataContainer</Name>
```
Comment (akeranen) [inline 10309.xml]: The name is quite generic for a relatively specific object

File: 10309.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      <Item ID="1">
+        <Name>BToothstate</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0-1</RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[BlueTooth State]]></Description>
```
Comment (akeranen) [inline 10309.xml]: More details on description would be useful. Is it on/off state? Maybe one of the reusable resources would be good here? Same comment applies to other state resources below.

File: 10309.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+      <Item ID="1">
+        <Name>BToothstate</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
```
Comment (akeranen) [inline 10309.xml]: Maybe boolean value is better here? Same comment applies to other state resources below

File: 10309.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        <RangeEnumeration>0-1</RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[Light State]]></Description>
+      </Item>
+      <Item ID="5">
+        <Name>UpTime</Name>
```
Comment (akeranen) [inline 10309.xml]: Would on-time re-usable resource (5852) fit here?

File: 10309.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        <Operations>RW</Operations>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
```
Comment (akeranen) [inline 10309.xml]: Probably numeric type is better

File: 10309.xml (hunk 1 above)
Comment (JiaDeng-Git) [reply 10309.xml:7]: We wanted to keep some information about the device, so I blurred the name.We're sorry about that.We will uphold an open and cooperative attitude to further improve the object.
Therefore, we change the name of the object to ShareParkingLock.
The lock is used to control each parking space and upload the lock information via wireless network.

File: 10309.xml (hunk 2 above)
Comment (JiaDeng-Git) [reply 10309.xml:25]: Yes,it's on/off state.
In the modified version, we deleted the resource.

File: 10309.xml (hunk 4 above)
Comment (JiaDeng-Git) [reply 10309.xml:60]: Yes, your suggestion is pretty good.
We want this time down to seconds.

File: 10309.xml (hunk 5 above)
Comment (JiaDeng-Git) [reply 10309.xml:64]: We add a little bit of notation to distinguish between the different times.However, after thinking about it, we also think that we should change to numeric.

**Comment by JiaDeng-Git** [comment]
We have submitted a new object, please review it.

File: 10309.xml (hunk 3 above)
Comment (JiaDeng-Git) [reply 10309.xml:21]: That's true,boolean value is better.

**Comment by jpradocueva** [comment]
This PR has been superseded by PR#76

**Pull request closed without merging**